common/src/node.rs

```diff
@@ -52,6 +52,8 @@ pub struct Node {
     /// We can only populate chunks which lie within a cube of populated nodes, so nodes on the edge
     /// of the graph always have some `None` chunks.
     pub chunks: Chunks<Chunk>,
+    /// Entities whose origins lie within this node
```

Comment:
I wonder whether we should be less specific and say something like "Entities currently associated with this node" here. On one hand, all our current entities should follow this rule, up to potentially some small buffer around each node in case floating point precision makes this ambiguous. On the other hand, this isn't a requirement at all, and we may still want to even tie entities without a `Position` component to the graph.

The main restriction is that the entity won't necessarily be loaded until the node is loaded, so for best results in the standard case, we do want this comment to be true.

Reply:
Moot due to switch to `GraphEntities`.